### ISSUE TITLE
Upcoming Appointment Details: VA Appointment - Accessibility Issues 

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#16675]
## What Happened?
([Ken] Note: This also applies to the other types of upcoming VA appointments: at an ATLAS location, using a VA device, at a VA location.)

**VoiceOver / TalkBak**

1. With the appropriate screen reader enabled on iOS and Android, the a11yHint for the back button is not being picked up 
     - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)
2. With the appropriate screen reader enabled on iOS and Android, the "...visit VA.gov" text is incorrectly picked up/pronounced

## Specs:
- Device: Android -Google Pixel 3XL(v.11) with TalkBack enabled
- Device: iOS - iPhone XR (v.14.2) with VoiceOver enabled
- User: marcy.nadeau@va.gov

## Steps to Reproduce
**VoiceOver / TalkBak**

For 2:
1. With the appropriate screen reader enabled on iOS or Android, navigate to a VA appointment page
2. Select the "...visit VA.gov" text in the "Need to cancel" section
3. Note that VA.gov is picked up/pronounced as Vaw [dot] gov on Android and as VA [dot] governor on iOS

## Desired Behavior
**VoiceOver / TalkBak**

For 2:
- With the appropriate screen reader enabled on iOS or Android, the "...visit VA.gov" text will be picked up/pronounced as VA [dot] gov

## Acceptance Criteria
- See desired behavior

## Bug Severity
- 4 - Low

## Linked Story (s)
- Story #13995

## Screen Shot (s) and Additional Information
2. Showing the affected areas 
![VA Appointment - Android Screen Reader.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/8e26abb2-f656-4cb4-bfbd-93783bfc9f19)

**Comment by digitalken** [comment]
PASS 1/8/2021 - VA.governor will be handled in another bug fix coming soon.
